docs/_documentations/mdt-eclipse-getting-started.md

```diff
@@ -32,14 +32,14 @@ To install Codewind for Eclipse, complete the following steps:
         5. When the installation is complete, restart Eclipse.
 4. Open the Codewind view.  Navigate to **Window** > **Show View** > **Other...** > **Codewind** > **Codewind Explorer**
 5. Codewind requires the installation of additional Docker images to run.  Double-click on the **Codewind** item in the Codewind Explorer view to complete the installation. The installation may take a few minutes to complete.
-6. Codewind creates a folder called `codewind-workspace` within your home directory (`C:\codewind-workspace` on Windows) to contain your projects.
+6. Codewind creates a codewind-data directory for storing its metadata. You should not save your projects to the codewind-data directory. 
```

Comment:
"should not" -> "must not"

Reply:
What about just "Do not"? I prefer the sentence in the imperative.
Ex: "Do not save your projects..."

Side note for Jake: Generally avoid "should." For example, avoid saying, "Click the button. The project should open." It's much better to say, "Click the button. The project opens." While there's the possibility that the project might not open, it's better if the instructions sound certain. :)

Reply:
Do not is fine, "should" "must" and "may" are common in developer docs because they are familiar from RFCs, the documents that lay out how the internet works. 
https://www.ietf.org/rfc/rfc2119.txt